ui/apps/dashboard/src/app/(organization-active)/(dashboard)/env/[environmentSlug]/apps/page.tsx

```diff
@@ -16,35 +24,66 @@ const AppInfo = () => (
     <TooltipContent
       side="right"
       sideOffset={2}
-      className="border-muted text-muted text-md mt-6 flex flex-col rounded-lg border p-0"
+      className="border-muted text-muted text-md mt-6 flex flex-col rounded-lg border p-0 text-sm"
     >
       <div className="border-b px-4 py-2 ">Apps map directly to your products or services.</div>
 
       <div className="px-4 py-2">
-        <Link href={'https://www.inngest.com/docs/apps'} className="text-md">
+        <NewLink href="https://www.inngest.com/docs/apps" size="small">
```

Comment:
Non-blocking for this PR: can we rename `NewLink` to `Link` and `Link` to `OldLink`? When doing UI work, I usually forget there's a new version of the component and I just type `Link`

Reply:
I hear you, I'll see if I can find some time between projects to tackle this. It's the same pattern as with Buttons, but ideally we would remove the old ones